.github/workflows/swift.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Xcode version check
```

Comment:
Make sure that you add the latest version from the develop branch here, I have updated it to use Xcode 12 for the time being.